sharktank/sharktank/examples/export_paged_llm_v1.py

```diff
@@ -98,7 +98,10 @@ def generate_params_json(hp, prefill_bs: list[int], decode_bs: list[int]):
             "prefill_batch_sizes": prefill_bs,
             "decode_batch_sizes": decode_bs,
             "transformer_block_count": hp.block_count,
-            "block_seq_stride": llama_config.block_seq_stride,
+            "paged_kv_cache": {
+                "block_seq_stride": llama_config.block_seq_stride,
+                "attn_head_count_kv": hp.attention_head_count_kv,
```

Comment:
Just FYI, in sharktank's [llm_configs.py](https://github.com/nod-ai/SHARK-Platform/blob/main/sharktank/sharktank/layers/configs/llm_configs.py#L67), hp.attention_head_count_kv==hp.attention_head_count.

Reply:
If you req vmfb and irpa for shortfin, is there a reason we rely on `generate_params_json` params instead of the `irpa` params? There are ways to load & access irpa params directly in sharktank. Do we want/have something similar in shortfin?

Reply:
Oh ty! i should probably take a deeper look then to figure this part out.